packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -1082,9 +1087,9 @@ describe("Data Store Context Tests", () => {
 			containerRuntime = {
 				IFluidDataStoreRegistry: registry,
 				on: (event, listener) => {},
-				logger: createChildLogger(),
+				baseLogger: createChildLogger(),
 				clientDetails: {},
-			} as ContainerRuntime;
+			} as unknown as ContainerRuntime;
```

Comment:
Weird that renaming that is triggering the compiler to behave differently.  Is that what's going on here, or something else? Oh well.

Reply:
Hmmm...It was complaining if I do "as ContainerRuntime" I need to set 148 parameters.